_tests/env/index.js

```diff
@@ -0,0 +1,6 @@
+//
+// Change which environment gets loaded by updating where the import points to.
+//
+import env from './production.js';
```

Comment:
> I'm not sure where the env files come into things - How is the env being detected?
> Do I need to change the env file in index.js to use locally?

Yes. This is the line hardcoded to prod. There's no mechanism for detecting environments; you change this line to run against local.